rust/processor/src/models/default_models/move_modules.rs

```diff
@@ -86,8 +90,12 @@ impl MoveModule {
     pub fn convert_move_module_bytecode(
         mmb: &MoveModuleBytecode,
     ) -> Option<MoveModuleByteCodeParsed> {
+        let abi = mmb.abi.as_ref();
+        if abi.is_none() {
+            return None;
+        }
         Some(Self::convert_move_module(
-            mmb.abi.as_ref().unwrap(),
+            abi.unwrap(),
             mmb.bytecode.clone(),
         ))
```

Comment:
nit; I think you can use a `map` or a match on `abi`

Reply:
true. done.